lib/tasks/rotate.rake

```diff
@@ -25,12 +24,13 @@ namespace :rotate do
   desc 'encrypt plain OTP secret key'
   task encrypt_otp: :environment do
     num_users = User.where.not(otp_secret_key: nil).count
-    progress = new_progress_bar('Users', num_users)
```

Comment:
ok will create a separate issue for this (since this change was just to quiet the `unused variable` error from rubocop)